jenkins/test-history/buckets.json

```diff
@@ -0,0 +1,11 @@
+{
+    "gs://kubernetes-jenkins/logs/": {
```

Comment:
I was thinking of extensibility.  Point of contact for the test job was something I had in mind.  The max job age is another great example.